docs/learn/submit-data.md

```diff
@@ -67,7 +67,7 @@ $\text{Total Fee} = \text{Gas Limit} \times \text{Gas Price}$
 
 The gas limit for a transaction is the maximum amount of gas that a user is
 willing to spend on a transaction. It is determined by both a static
-"fixed cost" and a dynamic cost based on the size of each blob involved
+fixed cost and a variable dynamic cost based on the size of each blob involved
```

Comment:
getting rid of the quotes is good, but imo [not blocking at all] variable and dynamic are redundant

Reply:
great call thanks Evan! https://github.com/celestiaorg/docs/pull/1051/commits/622c654a19540ae76d0ee7d1bf967a3a8c2b9dcb